Index: src/types/key-value.interface.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/types/key-value.interface.ts	(date 1565070152013)
+++ src/types/key-value.interface.ts	(date 1565070152013)
@@ -0,0 +1,4 @@
+export interface KeyValue<S = any, T = any> {
+    key: S;
+    value: T;
+}
Index: src/types/nested-string-map.interface.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/types/nested-string-map.interface.ts	(date 1602859207630)
+++ src/types/nested-string-map.interface.ts	(date 1602859207630)
@@ -0,0 +1,3 @@
+export interface NestedStringMap {
+    [key: string]: string | NestedStringMap;
+}
Index: src/types/string-map.interface.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/types/string-map.interface.ts	(date 1602764310205)
+++ src/types/string-map.interface.ts	(date 1602764310205)
@@ -0,0 +1,3 @@
+export interface StringMap<T = string> {
+    [s: string]: T;
+}
Index: src/types/size.interaface.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/types/size.interaface.ts	(date 1566630089016)
+++ src/types/size.interaface.ts	(date 1566630089016)
@@ -0,0 +1,4 @@
+export interface Size {
+    width: number;
+    height: number;
+}
Index: src/types/point.interaface.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/types/point.interaface.ts	(date 1566630089016)
+++ src/types/point.interaface.ts	(date 1566630089016)
@@ -0,0 +1,4 @@
+export interface Point {
+    x: number;
+    y: number;
+}
